src/app/beer_garden/api/stomp/transport.py

```diff
@@ -0,0 +1,212 @@
+
+    if conn.is_connected() and send_destination:
+        destination = send_destination
+        if request_headers and "reply-to" in request_headers:
+            destination = request_headers["reply-to"]
+
+        conn.send(body=message, headers=headers, destination=destination)
+
+
+class OperationListener(stomp.ConnectionListener):
+    def __init__(self, conn=None, send_destination=None):
+        self.conn = conn
+        self.send_destination = send_destination
+
+    def on_error(self, headers, message):
+        logger.warning(f"Error:\n\tMessage: {message}\n\tHeaders: {headers}")
+
+    def on_message(self, headers: dict, message: str):
+        """Handle an incoming message
+
+        Will parse the message as an Operation and attempt to route it. If the result of
+        the routing is truthy will send a response with the result.
+
+        If routing raised an exception an error response will be sent.
+
+        Args:
+            headers: Message header dict
+            message: The message body
+
+        Returns:
+            None
+        """
+        try:
+            operation = SchemaParser.parse_operation(message, from_string=True)
+
+            if hasattr(operation, "kwargs"):
+                operation.kwargs.pop("wait_timeout", None)
+
+            result = beer_garden.router.route(operation)
+
+            if result:
+                send(
+                    result,
+                    request_headers=headers,
+                    conn=self.conn,
+                    send_destination=self.send_destination,
+                )
+        except Exception as e:
+            send(
+                str(e),
+                request_headers=headers,
+                conn=self.conn,
+                send_destination=self.send_destination,
+            )
+            logger.warning(str(e))
+
+
+class Connection:
+    """Stomp connection wrapper
+
+    Args:
+        host:
+        port:
+        send_destination:
+        subscribe_destination:
+        ssl:
+        username:
+        password:
+
+    """
+
+    def __init__(
+        self,
+        host: str,
+        port: int,
+        send_destination: str = None,
+        subscribe_destination: str = None,
+        ssl=None,
+        username: str = None,
+        password: str = None,
+    ):
+        self.host = host
+        self.port = port
+        self.username = username
+        self.password = password
+        self.subscribe_destination = subscribe_destination
+        self.send_destination = send_destination
+        self.bg_active = True
+        self.conn = stomp.Connection(
+            host_and_ports=[(self.host, self.port)], heartbeats=(10000, 0)
+        )
+
+        if ssl and ssl.get("use_ssl"):
+            self.conn.set_ssl(
+                for_hosts=[(self.host, self.port)],
+                key_file=ssl.get("private_key"),
+                cert_file=ssl.get("cert_file"),
+            )
+
+    def connect(self) -> True:
```

Comment:
Kinda bold to type hint that it will always be `True`, instead of boolean

Reply:
Gotta be confident!